akka-stream/src/main/scala/akka/stream/impl/fusing/ActorGraphInterpreter.scala

```diff
@@ -520,7 +605,8 @@ final class GraphInterpreterShell(
     }
   }
 
-  override def toString: String = s"GraphInterpreterShell\n  ${assembly.toString.replace("\n", "\n  ")}"
+  // TODO: Fix debug string
+  override def toString: String = s"GraphInterpreterShell\n" //  ${assembly.toString.replace("\n", "\n  ")}"
```

Comment:
we don't have the assembly anymore. We can still report useful debug info here, so fix it.